### ISSUE TITLE
Screen reader is not announcing the selected option information for the 'search selection' control in Windows.

### ISSUE THREAD

**Issue by msftedad** [Semantic-Org/Semantic-UI-React#4519]
## Bug Report
Screen reader is not announcing the selected option information for the 'search selection' control in Windows. Screen reader dependent users will not be able to know the invoked action is performed successfully or not if screen reader is not announcing the selected option information.

### Steps
A clear and concise description of steps to reproduce the problem.

1. Turn on NVDA and Open URL: [Dropdown - Semantic UI React](https://react.semantic-ui.com/modules/dropdown/) in edge browser.
2. 'Drop down' page will appear.
3. Press tab key to reach till 'Search selection' control and activate it using enter key. Options will appear.
4. Select any option.
5. Verify whether screen reader is announcing the selected option information for the 'search selection' control or not.

### Expected Result
On selecting the option from the drop-down, screen reader should announce it as 'Afghanistan selected'.

### Actual Result
On selecting the option from the drop-down, Screen reader remains silent it doesn't announce any information.
Observation: When a country is previously selected and the user navigates back through the search selection, the screen reader does not announce which item is currently selected.

https://github.com/user-attachments/assets/e62fef6d-ab38-49f9-9038-84fca8cd0582

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/5ed3104c-795b-4a0f-9b6f-943d4745f7aa" />

### Version
3.0.0

### Testcase

https://react.semantic-ui.com/modules/dropdown/

**Comment by msftedad** [comment]
Hi Team, any update on this issue?
